test/e2e/app-dir/parallel-routes-not-found/parallel-routes-not-found.test.ts

```diff
@@ -21,5 +23,28 @@ describe('parallel-routes-and-interception', () => {
 
     // we also check that the #children-slot id is not present
     expect(await browser.hasElementByCssSelector('#children-slot')).toBe(false)
+
+    if (isPPR && !isNextDev) {
+      let $ = await next.render$('/')
+      expect($('title').text()).toBe('')
+
+      $ = await next.render$('/', null, {
+        headers: {
+          'User-Agent': 'Discordbot',
+        },
+      })
+      expect($('title').text()).toBe('layout title')
+    } else {
+      const $ = await next.render$('/')
+      expect($('title').text()).toBe('layout title')
+    }
+  })
+
+  it('should render the title once for the non-existed route', async () => {
+    const browser = await next.browser('/non-existed')
+    const titles = await browser.elementsByCss('title')
+
+    // FIXME: (metadata), the title should only be rendered once and using the not-found title
+    expect(titles).toHaveLength(3)
```

Comment:
We have another duplicated metadata issue which is not tight to this change, more like existing issue. But we can tackle that later